### DISCUSSION TITLE
Error at QuickAdd capture operations when using {{VALUE}}

### DISCUSSION THREAD

**Discussion by tjorrr** [chhoumann/quickadd#566] [Q&A] [answered]
Hi! 
I've been using QuickAdd as a part of my set-up for a while now, and I have not changed anything recently (apart from the latests update of Obsidian). Yet today, when I tried using my normal Capture scripts in which i need to enter a {{VALUE}} from my keyboard, I repeatedly got the same error:

<img width="543" alt="Screenshot 2023-09-08 at 14 24 05" src="https://github.com/chhoumann/quickadd/assets/92624286/79fa2c9d-3570-41ad-ad4e-4c07605292b6">

Basically, capture stopped working all of sudden. It allows me to capture already typed up text, and it also works when I take {{value}} out of the Capture format field, but it keeps flashing the same error as soon as I enter {{value}} back in. 

Any tips anyone?

**Comment by tjorrr** [comment, marked as answer]
Un-installing and installing the plug-in anew resolved the issue. I copied the previous settings from the .obsidian folder.

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that. Your fix was the right one: an error like this right after an Obsidian update usually means the plugin files were only partially updated, and reinstalling gives you a clean copy (and as you found, your settings survive in the `.obsidian` folder). Marking your comment as the answer and closing.
